sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -32,16 +32,14 @@ export type AppConfigurationApiVersion = (typeof AppConfigurationApiVersion)[key
 export const AppConfigurationApiVersion: {
     readonly Latest: "2022-11-01-preview";
     readonly Stable: "1.0";
-    readonly "1.0": "1.0";
-    readonly "2022-11-01-preview": "2022-11-01-preview";
```

Comment:
I think we should put these too?
cc @jeremymeng

Reply:
My feedback was about whether we should allow users to pass an api version.  if yes, we should make sure the errors are useful to users when they call new API that is only available in 2022-11-01-preview while passing in version of "1.0".

As for this particular, I believe they should be included for the pattern of `export type AppConfigurationApiVersion` +  `export const AppConfigurationApiVersion`

Reply:
or maybe not...I don't know the previous reason to have them

Reply:
Also, we are dropping "1.0" in the JS SDK

Reply:
The previous GA 1.4.1 SDK version doesn't have this.
Since we will support only the stable version in November GA, let's keep these until we go GA, so that users can use the preview and 1.0 stable versions. 
Does that sound good?

We'll remove these once we go GA in November and support the only recent stable service version.

@jeremymeng

Reply:
Sounds good to me. They'll be available for this October preview

Reply:
I've mentioned this to Jeff as well, he's fine with that

Reply:
Sounds good to me too.